src/Orc.FileAssociation/Models/ApplicationInfo.cs

```diff
@@ -42,9 +43,20 @@ public ApplicationInfo(Assembly assembly)
 
         public string Title { get; private set; }
 
-        public string Location { get; private set; }
+        public string Location
+        {
+            get
+            {
+                if (_location.EndsWith("dll"))
```

Comment:
EndsWithIgnoreCase

Reply:
Please add a note why we are doing this. It's probably because in .net core the exe is a stub and the actual entry assembly is a dll?

Reply:
Yes. If you put in a command in the registry path with dll, it will not start

Reply:
So please add that as a note in the code so we understand why we did this in 2 years time